subjects/java/raids/Jaikin/README.md

```diff
@@ -0,0 +1,37 @@
+## Implementation of Jaikin's Algorithm with Step-by-Step Animation
+
+### Objective
```

Comment:
I feel that it could be nice to add that the goal of the project is to implement a program that `... implement Chaikin's algorithm...`

I understood this only after viewing the video or reading the `audit`

Reply:
did you read the objective of the project ? it's clearly mentioned there !!

Reply:
In the `Objective` it is mentioned that the raid would be about implementing an algorithm, but not how it would be used: it could be a web app, a function, a CLI tool, ... 

It is clear after checking the audit and video that what it is expected is a program, but adding the information here as well could be nice

Reply:
This is what we have in the project objective ```The goal of this project is to implement Chaikin's algorithm and create a step-by-step animation of the process using a canvas``` we clearly mentioned that they have to impliment the chaikin's algorithm and they have to use canvas and java to do that so it's clear that it would be a program and not a web page or video or anything else

Reply:
This is the first result I found, looking for `canvas programming`: https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial

Without looking all the subject and the video provided, if I had to do this project, I could easily start looking on how to develop a web app and not a program (as it seems to be the case).

Reply:
I see that you forget that this project will build with java so if they have to search they have to search about canvas in java